sdk/avs/Azure.ResourceManager.Avs/src/Customized/CommonClusterProperties.cs

```diff
@@ -43,9 +44,11 @@ public partial class CommonClusterProperties
         /// </list>
         /// </para>
         /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
I am not quite sure what change is happening to this type. Could you specify what change happens in the new api-version in your spec for this model?

Reply:
CommonClusterProperties was deleted in the new api which caused the following compat issues:
`TypesMustExist : Type 'Azure.ResourceManager.Avs.Models.CommonClusterProperties' does not exist in the implementation but it does exist in the contract.

MembersMustExist : Member 'public Azure.ResourceManager.Avs.Models.CommonClusterProperties Azure.ResourceManager.Avs.Models.ArmAvsModelFactory.CommonClusterProperties(System.Nullable<System.Int32>, System.Nullable<Azure.ResourceManager.Avs.Models.AvsPrivateCloudClusterProvisioningState>, System.Nullable<System.Int32>, System.Collections.Generic.IEnumerable<System.String>)' does not exist in the implementation but it does exist in the contract.

CannotRemoveBaseTypeOrInterface : Type 'Azure.ResourceManager.Avs.Models.CommonClusterProperties' does not implement interface 'System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.Avs.Models.CommonClusterProperties>' in the implementation but it does in the contract.`

To resolve this I added back the CommonClusterProperties.cs,  CommonClusterProperties.Serialization.cs, and the method CommonClusterProperties in ArmAvsModelFactory